src/backend/distributed/citus--11.1-1.control

```diff
@@ -0,0 +1 @@
+requires = 'citus_columnar'
```

Comment:
what does this file do?

Reply:
this secondary control file is to make sure that citus_columnar must be installed first for citus version >= 11.1